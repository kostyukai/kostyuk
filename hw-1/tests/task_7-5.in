32187.5125
