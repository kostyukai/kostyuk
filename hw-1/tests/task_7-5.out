01000000000011011111101101110111000001100110011001100110011001100110011001100110
